scripts/cangen/cangen/can_generator.py

```diff
@@ -25,8 +25,9 @@
 MSG_REGISTRY_FILE_NAME = "_msg_registry.h"
 CAN_MESSAGES_FILE_NAME = "_can_messages.h"
 
-CAN_MESSAGES_TEMPLATE_FILENAME = "can_messages.h.jinja2"
-MSG_REGISTRY_TEMPLATE_FILENAME = "msg_registry.h.jinja2"
+TEMPLATE_FILE_NAMES = ["can_messages.h.jinja2", "msg_registry.h.jinja2"]
+
+PACKAGE_NAME = "cangen"
```

Comment:
use the `__package__` special variable. It holds the string name of the current package. Then you don't need a constant, just use it inline